packages/hub-threads-client/src/client.ts

```diff
@@ -66,6 +64,33 @@ Client.prototype.getThread = async function (
   })
 }
 
+// Private listDBs method
+const oldListDBs = Client.prototype.listDBs
+
+/**
+ * Lists all known DBs.
+ */
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
```

Comment:
Note the `ts-ignore`, because this change is interface-breaking re: upstream `@textile/threads-client` `listDBs`.